tests/cypress/e2e/issues_prs2/issue_8952_interpolation_impossible.js

```diff
@@ -0,0 +1,179 @@
+            cy.createZipArchive(dirToArchive, archivePath);
+            cy.createTaskFromArchive(taskName, labelName, archiveName);
+
+            cy.goToTaskList();
+            cy.openTaskJob(taskName);
+            cy.url().should('contain', 'jobs').then((url) => {
+                const last = url.lastIndexOf('/');
+                jobID = parseInt(url.slice(last + 1), 10);
+            }).then(() => {
+                // Remove all annotations and draw a track rect
+                const wrap = (shape, frame) => shapeToPayload(shape, frame, 'rectangle');
+                const shape0 = createRectangleShape2Points;
+                const shape1 = translateShape(shape0, delta, 'x');
+                const shape2 = translateShape(shape1, delta, 'y'); // TODO: fix coords, rect flies away
+                const track = makeTrack([
+                    wrap(shape0, 0),
+                    wrap(shape1, 2),
+                    wrap(shape2, 4),
+                ], 0, labelName);
+                cy.headlessCreateObjects([track], jobID);
+            });
+        });
+
+        beforeEach(() => {
+            // Restore all frames by patching deleted_frames to []
+            cy.intercept('PATCH', '/api/jobs/**/data/meta**').as('patchMeta');
+            cy.headlessRestoreAllFrames(jobID);
+            cy.wait('@patchMeta');
+
+            // Get job meta updates from the server and reload page to bring changes to UI
+            cy.intercept('GET', '/api/jobs/**/data/meta**').as('getMeta');
+            cy.reload();
+            cy.wait('@getMeta');
+
+            cy.saveJob();
+            cy.clickFirstFrame();
+        });
+        it('Delete interpolated frames 0, 2, 4. Error should not appear', () => {
+            cy.on('uncaught:exception', (err) => {
+                const expectedMsg = 'No one left position or right position was found. Interpolation impossible. Client ID:';
+                expect(err.message).to.include(expectedMsg); // Exclude familiar error
+                throw err;
+            });
+
+            // Delete frames 0, 2, 4. Watch out for errors
+            cy.clickFirstFrame();
+            cy.checkFrameNum(0);
+            cy.clickDeleteFrameAnnotationView();
+            cy.checkFrameNum(1);
+            cy.goToNextFrame(2);
+            cy.clickDeleteFrameAnnotationView();
+            cy.checkFrameNum(3);
+            cy.goToNextFrame(4);
+            cy.clickDeleteFrameAnnotationView();
+
+            // There should be no objects on the deleted frame
+            cy.get('.cvat_canvas_shape').should('not.exist');
+            cy.clickSaveAnnotationView();
+
+            // This might pop up after deleting or saving (on firefox e.g.)
+            cy.get('.ant-notification-notice-error').should('not.exist');
+
+            // Reopening a task with bad metadata might throw an exception that we can catch
+            cy.goToTaskList();
+            cy.openTaskJob(taskName);
```

Comment:
cy.reload() and corresponding interceptor (e.g. annotations receiving) to make sure that the page was updated?